node/src/components/fetcher.rs

```diff
@@ -450,7 +471,7 @@ where
                     self.metrics.found_in_storage.inc();
                     self.got_from_storage(item, peer)
                 }
-                None => self.failed_to_get_from_storage(effect_builder, id, peer),
+                _ => self.failed_to_get_from_storage(effect_builder, id, peer),
```

Comment:
Why not `None`?

Reply:
Good catch!  Fixed in a2dd9b2bed8f877e50e5ab7d6aeb49483165f3e3